test/src/api/commands/element/testClearValue.js

```diff
@@ -135,32 +135,49 @@ describe('clearValue', function() {
     }).then((client) => {
       const bArr = Array(6).fill(Key.BACK_SPACE);
       const str = bArr.join('');
+      let sendKeysMockCalled = false;
   
       MockServer.addMock(
         {
           url: '/session/13521-10219-202/element/5cc459b8-36a8-3042-8b4a-258883ea642b/clear',
+          response: {value: null}
+        },
+        true
+      );
+
+      MockServer.addMock(
+        {
+          url: '/session/13521-10219-202/element/5cc459b8-36a8-3042-8b4a-258883ea642b/property/value',
           response: {value: 'sample'}
         },
         true
       );
+
   
       MockServer.addMock(
         {
           url: '/session/13521-10219-202/element/5cc459b8-36a8-3042-8b4a-258883ea642b/value',
           method: 'POST',
           postdata: {
-            str,
+            text : str,
             value: bArr
           },
           response: {
-            value: null
+            sessionId: '13521-10219-202',
+            status: 0
+          },
+          onRequest() {
+            sendKeysMockCalled = true;
           },
           statusCode: 200
         },
         true
       );
+      
   
-      client.api.clearValue('#webdriver', function (result) {
+      client.api.sendKeys('css selector', '#webdriver', bArr, function callback(result) {
+        assert.strictEqual(sendKeysMockCalled, true);
```

Comment:
Why are you calling `.sendKeys()` command directly? We just want to call `.clearValue` command from outside and then we want to test if the internal `element.sendKeys()` (which you specified in method-mappings.js file) was called or not if the `element.getProperty('value')` command still returns some value.

Reply:
Hi @garg3133 ,
for some reason, the post request was not being sent when I tried checking the logs
So I looked into the codebase to see how keys were being sent in a test and I saw this method
I tried doing it by using .sendKeys() and it worked

Reply:
` client.api.sendKeys('css selector', '#webdriver', bArr, function callback(result) {
       
      }).clearValue('css selector', '#webdriver', function (result) {
        assert.strictEqual(result.value, null);
        assert.strictEqual(sendKeysMockCalled, true);
        // ensures fallback is working
      });`

this is working as well

Reply:
> Hi @garg3133 , for some reason, the post request was not being sent when I tried checking the logs So I looked into the codebase to see how keys were being sent in a test and I saw this method I tried doing it by using .sendKeys() and it worked

Are you still working on this issue ?

Reply:
yes I am working on this. had been sick that's why the delay
apologies

Reply:
Hi, just wanted to make sure that this is all still being worked on. I could land an eye if you need. (not sure that I can be of a great help but I can still try 😬)